code/go/0chain.net/smartcontract/storagesc/blobber.go

```diff
@@ -75,13 +74,27 @@ func (sc *StorageSmartContract) updateBlobber(t *transaction.Transaction,
 		return fmt.Errorf("can't get or decode saved blobber: %v", err)
 	}
 
+	if savedBlobber.BaseURL != blobber.BaseURL {
```

Comment:
In line 72
```go
savedBlobber, err := sc.getBlobber(blobber.ID, balances)
```
Is this MPT access needed? I think blobber `blobber.Id` might already have been retrieved [further up the stack](https://github.com/0chain/0chain/blob/feature/remove-all-blobbers/code/go/0chain.net/smartcontract/storagesc/blobber.go#L835), so could we pass this in as a parameter?

Reply:
Yes, the blobber should have been acquired, we don't need to get from MPT again.